### ISSUE TITLE
Won't Download images from Furaffinity

### ISSUE THREAD

**Issue by Dragonrider900** [Simon-Tesla/RaccoonyWebEx#112]
**Describe the bug**
Click the download button to download an image but it says that it is downloading and no image downloading.

**To Reproduce**
1. Click on the download button.
2. See button says it's downloading.
3. doesn't download the image.
4. Refresh the page and try again
5. Tried it with other furaffinity images.

**Expected behavior**
To click on the download button and download the image to Raccoony folder

**Screenshots**
[Raccoony error](https://user-images.githubusercontent.com/9660200/132886876-4f796435-b555-4172-956c-31ead75975ac.gif)

**Environment (please complete the following information):**
 - Windows 10
 - Waterfox
 - G3.2.5 
 - 1.2.5

**Additional context**
Add any other context about the problem here. Include:
 - The address of a site that reproduces the bug, if applicable
 - Any settings you may have customized from the defaults (if possible, please export a copy of your settings file from the global Raccoony options page and attach it to the bug)

**Comment by Simon-Tesla** [comment]
I don't repro in Firefox, and I don't test in Waterfox, so I'm not sure what would be going on here. This happens for any pictures? Have you changed any of the download-related options?

**Comment by Eupeptic1** [comment]
I tested Raccoony 1.2.5 a couple of days ago, and i did not repro this.  Testing was on Firefox 78.13.0esr desktop and Chrome 93.0.4577.63 desktop, both on Debian Linux, 64-bit.

I also can't repro on Waterfox G3.2.5 (64-bit), desktop, on Debian 64-bit.  Waterfox allows both Firefox and Chrome extensions.  I installed the Firefox version of Raccoony 1.2.5 and was able to download three random images from the recent submissions/"browse" section of FA.  I then uninstalled Firefox Raccoony, installed Chrome Raccoony 1.2.5, and tested again, and images downloaded OK.

I also tried downloading the specific submission that the reporter had in the screenshot ( 40483667 ), and it downloaded OK in Waterfox with Firefox Raccoony.

Just for fun, I arranged for `~/Downloads/raccoony` to live on a tiny partition (10 megabytes) that was nearly full, and then tried downloading a few images in Waterfox with Firefox Raccoony, just to see if it was possibly a disk space issue.  Once I filled up the partition, clicking "Download" on a new image resulted in the Download button immediately changing to say "Download error"; it did not hang Raccoony or Waterfox.  So I can't repro there, either.
